CMakeLists.txt

```diff
@@ -20,3 +29,16 @@ set_property(TARGET oeedger8r PROPERTY POSITION_INDEPENDENT_CODE on)
 enable_testing()
 add_subdirectory(test)
 
+option(CODE_COVERAGE "Enable code coverage testing" OFF)
+if(CODE_COVERAGE)
+  message("Building for code coverage.")
+  target_compile_options(oeedger8r PRIVATE -g -O0 -coverage -fprofile-arcs
+                                           -ftest-coverage)
+  target_link_libraries(oeedger8r PRIVATE -lgcov)
```

Comment:
This is gcc-specific. Should we add check?

Reply:
Done.